### PR TITLE
Use `service_name` instead of `app_id`

### PR DESCRIPTION
**Pull request by sschneid**

### CODE REVIEW

File: shakedown/dcos/package.py [hunk 1]
```diff
@@ -126,33 +134,33 @@ def install_package_and_wait(
...
 
-def package_installed(package_name, app_id=None):
+def package_installed(package_name, service_name=None):
     """ Check whether thea package package_name is currently installed.
 
         :param package_name: package name
         :type package_name: str
-        :param app_id: app_id
-        :type app_id: str
+        :param service_name: app_id
```
Comment (colin-msphere) [inline shakedown/dcos/package.py]: This should probably not be `app_id`.

File: shakedown/dcos/package.py [hunk 2]
```diff
@@ -100,7 +108,7 @@ def install_package(
         future = now + timeout_sec
 
         while now < future:
-            if shakedown.get_service(package_name):
+            if shakedown.get_service(service_name):
```
Comment (colin-msphere) [inline shakedown/dcos/package.py]: Did this mean to get changed? Thus far, everything has been previously called `app_id`, whereas this one was `package_name`.

File: shakedown/dcos/package.py (hunk 1 above)
Comment (sschneid) [inline shakedown/dcos/package.py]: Oops, will fix.

File: shakedown/dcos/package.py (hunk 2 above)
Comment (sschneid) [inline shakedown/dcos/package.py]: Yep -- this was broken when waiting on services deployed with custom service names.

**Comment by sschneid** [comment]
@mesosphere-ci retest this please

**Pull request merged**